### PR TITLE
Shipping rates API

### PR DESCRIPTION
**Pull request by justnero**

### CODE REVIEW

File: openapi/components/schemas/ShippingTypes/rebilly.yaml [hunk 1]
```diff
@@ -3,6 +3,11 @@ allOf:
   - $ref: ../Shipping.yaml
   - type: object
     properties:
+      shippingRateId:
```
Comment (creocoder) [inline openapi/components/schemas/ShippingTypes/rebilly.yaml:6]:
```suggestion
      rateId:
```

File: openapi/components/schemas/ShippingTypes/rebilly.yaml [hunk 2]
```diff
@@ -3,6 +3,11 @@ allOf:
   - $ref: ../Shipping.yaml
   - type: object
     properties:
+      rateId:
+        description: The shipping rate ID. If unset – cheapest shipping rate among applicable would be chosen.
+        nullable: true
+        allOf:
+          - $ref: ../../ResourceId.yaml
```
[the remark this answers is not in the document]
Comment (Boydbueno) [reply openapi/components/schemas/ShippingTypes/rebilly.yaml]: Thanks. Resolved it.

**Review by Boydbueno — DISMISSED** [review]

File: openapi/components/schemas/ShippingTypes/rebilly.yaml [hunk 3]
```diff
@@ -3,6 +3,11 @@ allOf:
   - $ref: ../Shipping.yaml
   - type: object
     properties:
+      rateId:
+        description: The shipping rate ID. If unset – cheapest shipping rate among applicable would be chosen.
```
Comment (adamaltman) [inline openapi/components/schemas/ShippingTypes/rebilly.yaml:7]:
```suggestion
        description: The shipping rate ID. If unset the cheapest applicable shipping rate is chosen.
```

File: openapi/components/schemas/Storefront/Purchase.yaml [hunk 4]
```diff
@@ -56,6 +56,10 @@ properties:
     nullable: true
     allOf:
       - $ref: ../Contact/ContactObject.yaml
+  shippingRateId:
+    description: The shipping rate ID. If unset – cheapest shipping rate among applicable would be chosen.
```
Comment (adamaltman) [inline openapi/components/schemas/Storefront/Purchase.yaml:60]:
```suggestion
    description: The shipping rate ID. If unset the cheapest applicable shipping rate is chosen.
```

**Review by adamaltman — CHANGES_REQUESTED** [review]

File: openapi/components/schemas/Common/CommonOrderPreview.yaml [hunk 5]
```diff
@@ -36,6 +36,10 @@ properties:
     nullable: true
     allOf:
       - $ref: ../Contact/ContactObject.yaml
+  shippingRateId:
+    description: The shipping rate ID. If unset – cheapest shipping rate among applicable would be chosen.
```
Comment (vykhrystiuk) [inline openapi/components/schemas/Common/CommonOrderPreview.yaml:40]:
```suggestion
    description: The shipping rate ID. If unset the cheapest applicable shipping rate is chosen.
```

https://github.com/Rebilly/api-definitions/pull/644#discussion_r791964287

**Review by adamaltman — APPROVED** [review]

**Comment by Boydbueno** [comment]
Please review

**Review by ashkarpetin — APPROVED** [review]

**Pull request merged**